constants.js

```diff
@@ -0,0 +1,6 @@
+
+module.exports = {
+
+    DATABASE_ERROR_RESPONSE: "An unexpected database error occurred. Please ask an administrator to check atlas logs at YYYY-DD-MM HH:MM:SS for more information"
```

Comment:
This is great; however, I've missed where "YYYY-DD-MM HH:MM:SS" is being converted to the server's current timestamp. Is that happening somewhere? Or are we sending this message unchanged?

Reply:
Distant screaming: NOOOOOOOOOOOOOOO!!!!!!

Updated the PR :)))

Reply:
I hate to nitpick, but you've got dozens of cookie-cutter lines that require specific knowledge of the constant's format, do fancy slicing & dicing of dates, and require `util` and `constants.js`. Instead, how about eliminating `constants.js` and, instead, adding this to our `util.js`?

```node
genericDatabaseErrorMessage: function() {
	return util.format(
		"An unexpected database error occurred. Please ask an administrator to check atlas logs at %s for more information",
		dateformat(new Date(), 'mm/dd/yyyy h:MM:ss TT')
	)
}
```

Then you only need to require our `util.js` and the entries can go from this:

```node
return res.status(500).send({ message: util.format(constants.DATABASE_ERROR_RESPONSE, new Date().toISOString().slice(0, 19).replace('T', ' ')) });
```

to this:

```node
return res.status(500).send(util.genericDatabaseErrorMessage());
```

Reply:
Much better! Updated the PR :)